src/middlewares/basicAuthMiddleware.js

```diff
@@ -102,15 +93,15 @@ const getHashFromHeaders = (headers, opts) => {
   return buff.toString('utf-8');
 };
 
-const validateBasicAuth = (hash, siteId, clientObject, opts) => {
+const validateBasicAuth = (hash, clientObject, opts, siteId = undefined) => {
   const site = Object.keys(clientObject).find(clientCode => {
```

Comment:
@jpaylaga  I just realized that even with siteId as params, we have been doing the loop anyways right ? but should be fine not so much of a deal to being with

Reply:
Yep. That is correct.